git/git.go

```diff
@@ -147,10 +160,10 @@ func fatalGit(s ssh.Session, err error) {
 	msg := err.Error()
 	pktLine := fmt.Sprintf("%04x%s\n", len(msg)+5, msg)
 	_, _ = s.Write([]byte(pktLine))
-	s.Exit(1)
+	s.Exit(1) // nolint: errcheck
```

Comment:
Note: a slightly easier way to communicate we intentionally ignore a returned error in these situations, is following what we do in the line above:

```go
_ = s.Exit(1)
```

This also satisfies the linter. The same goes for `srv.Close()` below.

Reply:
indeed